be/src/util/slice.h

```diff
@@ -273,6 +273,13 @@ struct Slice {
         }
         return buf;
     }
+
+    // X is (maybe) a truncated prefix of string X'
+    // Y is (maybe) a truncated prefix of string Y'
+    // return true only if we can determine that X' is strictly less than Y'
+    // based on these maybe truncated prefixes
+    static bool lhs_is_strictly_less_than_rhs(Slice X, bool X_is_truncated, Slice Y,
```

Comment:
the comment is confusing  and the parameters are not explained well
what is X and what is x_is_truncated.
it is better to show some examples to describe the behavior of this function.